.gitmodules

```diff
@@ -18,7 +18,10 @@
 	url = https://github.com/meta-rust/meta-rust.git
 [submodule "layers/meta-up-board"]
 	path = layers/meta-up-board
-	url = https://github.com/AaeonCM/meta-up-board.git
+	url = https://github.com/up-division/meta-up-board
 [submodule "contracts"]
 	path = contracts
 	url = https://github.com/balena-io/contracts.git
+[submodule "layers/meta-balena-up-board/recipes-kernel/pinctrl-upboard"]
```

Comment:
Why is this a submodule? The meta-balena-up-board is a regular directory and not a submodule